collectors/audit/nfrr/README.md

```diff
@@ -0,0 +1 @@
+The README is in the [gh-pages](https://github.com/capitalone/Hygieia/blob/gh-pages/pages/hygieia/collectors/build/nfrr-audit.md) branch. Please update it there.
```

Comment:
Suggested Additions for the nfrr-audit.md

# Purpose

The No Fear Release (NFRR) Collector aggregates Json responses from a variety of Hygieia Audit APIs.
The NFRR Collector sources information from Peer Review, Static Code Analysis, Static Security Analysis, and Performance Audit APIs into a Json response.

## Dependencies/Assumptions

## Collector
NFRR Collector outputs the following Json schema
Portfolio,Product,Component,Product_Owner,Portfolio_Owner,Audit_Type,Collection_Status,Audit_Status,URL,Reason for Error

Json Response Example:
<json example here>

- Audit Type
  - Specified type in the dashboardReview endpoint (see OpenApi docs)

- Collection Status
  - NOT CONFIGURED - Collector has not been setup  
  - OK - Collector ran without errors
  - FAIL - Collector encountered error(s), such as connection time outs, cron or property issues

## API

# Roadmap Items
- Add other Audit APIs to the NFRR Collector, such as Traceability to build out full representation of the Audit API family.